### PR TITLE
*: fix `TestLogicalOverflow`

### LINKED ISSUE #6277: TestLogicalOverflow failed
### Which jobs are failing
--- FAIL: TestLogicalOverflow (9.53s)
    global_tso_test.go:169: 
        	Error Trace:	/home/runner/work/pd/pd/tests/server/tso/global_tso_test.go:169
        	            				/home/runner/work/pd/pd/tests/server/tso/global_tso_test.go:175
        	Error:      	"1.567606ms" is not greater than or equal to "50ms"
        	Test:       	TestLogicalOverflow

### CI link
https://github.com/tikv/pd/actions/runs/4626540326/jobs/8183394949?pr=6247

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve?

Issue Number: Close #6277.

### What is changed and how does it work?
Even if the `updateInterval` is 50ms, we can still encounter the [problem](https://github.com/tikv/pd/compare/master...rleungx:fix-unstable-test2?expand=1#diff-d27962ba195b8fe8deead66b845c96f6f47bcc2ff5159ade2f236075301b93fdL168). This PR adds one more request to make sure the test can be passed every time. 

```commit-message
```

### Check List

Tests

- Unit test

### Release note

```release-note
None.
```

### CODE REVIEW

File: tests/server/tso/global_tso_test.go [hunk 1]
```diff
@@ -163,12 +163,13 @@ func TestLogicalOverflow(t *testing.T) {
 			re.NoError(tsoClient.Send(req))
 			_, err = tsoClient.Recv()
 			re.NoError(err)
+			if i == 1 {
+				// the 2nd request may (but not must) overflow, as max logical interval is 262144
+				re.Less(time.Since(begin), updateInterval+20*time.Millisecond) // additional 20ms for gRPC latency
```
Comment (binshi-bing) [inline tests/server/tso/global_tso_test.go:168]: Is it possible that "+20*time.Millisecond) // additional 20ms for gRPC latency" causes random failure?

**Review by binshi-bing — APPROVED** [review]

File: tests/server/tso/global_tso_test.go (hunk 1 above)
Comment (rleungx) [reply tests/server/tso/global_tso_test.go:168]: Yes, good catch.

Comment (rleungx) [reply tests/server/tso/global_tso_test.go:168]: But it's a local environment, so the latency might not be that slow. We can solve it after we encounter it.

**Review by JmPotato — APPROVED** [review]

**Pull request merged**